.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,5 +6,5 @@ contact_links:
     url: https://stackoverflow.com/questions/tagged/twilio-go+or+twilio+go
     about: Ask questions on Stack Overflow
   - name: Documentation
-    url: https://github.com/twilio/twilio-go#getting-started
+    url: https://pkg.go.dev/github.com/twilio/twilio-go?tab=versions
```

Comment:
I think this is generated by librarian.

Reply:
I changed this link recently before the pilot launch. [Here](https://code.hq.twilio.com/twilio/librarian/blob/36c100847004b517943eb50f8ea9ca663b1aad17/librarian/library.py#L159) is where you can change the url in librarian.